src/views/Upload.vue

```diff
@@ -39,27 +52,87 @@ import UploadTab from '@/components/Upload/UploadTab.vue';
 import UploadFile from '@/components/Upload/UploadFile.vue';
 import UploadText from '@/components/Upload/UploadText.vue';
 
+import axios from 'axios';
+
 export default {
 	components: {
 		Navigation,
 		UploadTab,
 		UploadFile,
-		UploadText,
+		UploadText,	
 	},
 	data: function () {
 		return {
 			selectedTab: '',
 			tabs: ['Nahrať súbory', 'Vložiť text'],
+			files: [],
+			text: '',
 		};
 	},
+	methods: {	
+		handleFiles(filesArray){
+			Object.values(filesArray).forEach(file => {
+				this.files.push(file);     
+			});
+		},
+		handleText(text){
+			this.text = this.text + text;
+			console.log(this.text)
+		},
+		removeFiles(key){
+			this.files.splice( key, 1 );
+		},
+		submitFile() {
+			let formData = new FormData();
+
+			this.files.forEach(file => {
```

Comment:
Can we just differ using what tab is currently selected? If we're in files, then do all the stuff with files, and check for length and add set alert..


```js

create headers variable
if currenttab = files 

.... add file to formdata

... check if files not empty, 
   if so, alert

... set headers to formdata

else 

... check whether text is not empty

... set headers to text plain

end of if block

set loader to true
return axios request with formdata and headers